packages/block-library/src/columns/editor.scss

```diff
@@ -41,24 +48,17 @@
 
 		// Adjust the individual column block.
 		> [data-type="core/column"] {
-			display: flex;
-			flex-direction: column;
-			flex: 1;
-
-			// The Column block is a child of the Columns block and is mostly a passthrough container.
-			// Therefore it shouldn't add additional paddings and margins, so we reset these, and compensate for margins.
-			// @todo In the future, if a passthrough feature lands, it would be good to apply these rules to such an element in a more generic way.
-			> .editor-block-list__block-edit > div > .editor-inner-blocks {
-				margin-top: -$block-padding - $block-padding;
-				margin-bottom: -$block-padding - $block-padding;
-			}
-
+			@include flex-full-height();
+			// Enable vertical alignment preview
 			> .editor-block-list__block-edit {
-				margin-top: 0;
-				margin-bottom: 0;
+				@include flex-full-height();
+				> div[data-block],
+				.block-core-columns {
+					@include flex-full-height();
```

Comment:
Minor: We're using `@include flex-full-height();` in three places here. Provided that it's required for all four selectors, we might want to consider combining the selectors so we just define it once.

```
> [data-type="core/column"],
> .editor-block-list__block-edit,
> .editor-block-list__block-edit > div[data-block],
> .editor-block-list__block-edit .block-core-columns {
  @include flex-full-height();
}
```

Reply:
As long as specificity doesn't bit us (and it looks good) we should be ok to do this. Thank you!

Reply:
I tried the code you suggested but it didn't quite work. 

I've amended up pulling all the selectors onto one line. This results in some duplication but it is perhaps clearer and easier to group the selectors into a single unit of code.